addons/website/static/src/scss/website.edit_mode.scss

```diff
@@ -233,6 +233,12 @@ body.editor_enable {
         > * {
             pointer-events: none;
         }
+        // REMOVEME in master
+        // BS4 adds pointer-events: auto on this which breaks the editor.
+        // BS5 doesn't do that so this can safely be removed in master.
+        .stretched-link::after {
+            pointer-events: auto;
+        }
```

Comment:
> BS4 adds pointer-events: auto on this which breaks the editor

That's why you force it even more to `auto`? ^^

Reply:
@detrouxdev Didn't we conclude it should be `None` ?
Reminder: We thought about removing the `>` from `> *` and saw that pseudo element are not targetted by wild card selector: https://stackoverflow.com/a/24794612/3272802

Reply:
It should just be pointer-events none on stretched-link::after as explained here: https://github.com/odoo/odoo/pull/82222#discussion_r932267173 (BS4 resetting it back to auto automatically).

Indeed the *, ::after, ::before { pointer-events: none; } solution was right too (handling all potential custo), whatever is fine. Just the explanation was not right :wink:

Reply:
Oops, my bad! It should be `pointer-events: none;`, I fixed it. Since the issue doesn't occur on master, I think it might be better if we stay specific.